Wrappers/Python/cil/optimisation/algorithms/SIRT.py

```diff
@@ -17,38 +17,78 @@
 from cil.optimisation.algorithms import Algorithm
 from cil.optimisation.functions import IndicatorBox
 from numpy import inf
+import numpy
 import warnings
 
-class SIRT(Algorithm):
+class SIRT(Algorithm):    
 
-    r'''Simultaneous Iterative Reconstruction Technique
+    r"""Simultaneous Iterative Reconstruction Technique 
+
+    Simultaneous Iterative Reconstruction Technique (SIRT) solves
+    the following problem
+
+    .. math:: A x = b
+
+    The SIRT algorithm is 
+
+    .. math:: x^{k+1} =  \mathrm{proj}_{C}( x^{k} + D ( A^{T} ( M * (b - Ax) ) ) ),
+
+    where :math:`M = A*\mathbb{1}`, :math:`D = A^{T}\mathbb{1}`, :math:`\mathbb{1}` is a :code:`DataContainer` of ones
+    and :math:`\mathrm{prox}_{C}` is the projection over a set :math:`C`.
+
+    Parameters
+    ----------
+
+    initial : DataContainer, default = None
+              Starting point of the algorithm, default value = Zero DataContainer 
+    operator : LinearOperator
+              The operator A.
+    data : DataContainer
+           The data b.
+    lower : :obj:`float`, default = None
+            Lower bound constraint, default value = :code:`-inf`.
+    upper : :obj:`float`, default = None
+            Upper bound constraint, default value = :code:`-inf`.
+    constraint : IndicatorBox function, default = None
+                 Enforce box constraint using the :class:`.IndicatorBox` function.
+    kwargs:
+        Keyword arguments used from the base class :class:`.Algorithm`.    
+
+    Note 
+    ----
+    
+    If :code:`constraint` is passed, it should be an :class:`.IndicatorBox` function, 
+    and in that case :code:`lower` and :code:`upper` inputs are ignored. 
     
-    Problem: 
+    If :code:`constraint` is passed, then :code:`lower` and :code:`upper` are looked at, 
```

Comment:
Should it be
If :code:`constraint` is NOT passed
 ?

Reply:
fixed